kevm-pyk/src/kevm_pyk/kevm.py

```diff
@@ -20,8 +18,6 @@
 from pyk.prelude.string import stringToken
 from pyk.utils import unique
 
-from .utils import add_include_arg
```

Comment:
I believe this was the only use of this function, can we remove ti from `utils.py` too?

Reply:
There's one more reference to it in `__main__.exec_prove` unfortunately.